src/cli/commands/floodsub/pub.js

```diff
@@ -0,0 +1,28 @@
+'use strict'
+
+const utils = require('../../utils')
+const debug = require('debug')
+const log = debug('cli:floodsub')
+log.error = debug('cli:floodsub:error')
+
+module.exports = {
+  command: 'pub <topic> <message>',
```

Comment:
Sorry if I missed that (and I do like the full name convention). If you're good with it, I'll make that change to sync up with what you folks already agreed on.